pkg/apis/applicationmonitoring/v1alpha1/applicationmonitoring_types.go

```diff
@@ -23,6 +23,8 @@ type ApplicationMonitoringSpec struct {
 	BlackboxTargets                  []BlackboxTarget `json:"blackboxTargets,omitempty"`
 	AdditionalScrapeConfigSecretName string           `json:"additionalScrapeConfigSecretName,omitempty"`
 	AdditionalScrapeConfigSecretKey  string           `json:"additionalScrapeConfigSecretKey,omitempty"`
+	PrometheusRetention              string           `json:"prometheusRetention"`
```

Comment:
Any reason why not to add `omitempty` to this one too? This is to make the field optional.

Reply:
Good point. Would that need a change to setting the default of `15d` here as well? https://github.com/integr8ly/application-monitoring-operator/pull/56/files#diff-3d0eb3cd57e9f53087a8908bb217cb6eR130

Reply:
I don't think so because the field is still present in the struct and initialized to an empty string.

Reply:
Just checked it. All good.
Pushed the suggested change